pkgs/applications/networking/instant-messengers/feishu/default.nix

```diff
@@ -179,7 +183,7 @@ stdenv.mkDerivation {
       wrapProgram $executable \
         --prefix XDG_DATA_DIRS    :  "$XDG_ICON_DIRS:$GSETTINGS_SCHEMAS_PATH" \
         --prefix LD_LIBRARY_PATH  :  ${rpath}:$out/opt/bytedance/feishu:${addOpenGLRunpath.driverLink}/share \
-        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}" \
+        ${nativeWaylandSupport}
```

Comment:
Missing `\`.

We'd better just remove it and leave a comment

Reply:
Yeah, I agree with Aleksanaa - this breaks the following line due to the use of `optionalString` + newline. The comment should not go in the middle of the `wrapProgram` command either, somewhere before is fine.